descriptions/api.github.com/api.github.com.json

```diff
@@ -597,6 +597,153 @@
+                    "$ref": "#/components/schemas/hook-delivery-item"
+                  }
+                },
+                "examples": {
+                  "default": {
+                    "$ref": "#/components/examples/hook-delivery-items"
+                  }
+                }
+              }
+            }
+          },
+          "400": {
+            "$ref": "#/components/responses/bad_request"
+          },
+          "422": {
+            "$ref": "#/components/responses/validation_failed"
+          }
+        },
+        "x-github": {
+          "githubCloudOnly": false,
+          "enabledForGitHubApps": false,
+          "previews": [
+
+          ],
+          "category": "apps",
+          "subcategory": "webhooks"
+        }
+      }
+    },
+    "/app/hook/deliveries/{delivery_id}": {
+      "get": {
+        "summary": "Get a delivery for an app webhook",
+        "description": "Returns a delivery for the webhook configured for a GitHub App.\n\nYou must use a [JWT](https://docs.github.com/apps/building-github-apps/authenticating-with-github-apps/#authenticating-as-a-github-app) to access this endpoint.",
+        "tags": [
+          "apps"
+        ],
+        "operationId": "apps/get-webhook-delivery",
+        "externalDocs": {
+          "description": "API method documentation",
+          "url": "https://docs.github.com/rest/reference/apps#get-a-delivery-for-an-app-webhook"
+        },
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/delivery-id"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Response",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/hook-delivery"
+                },
+                "examples": {
+                  "default": {
+                    "$ref": "#/components/examples/hook-delivery"
+                  }
+                }
+              }
+            }
+          },
+          "400": {
+            "$ref": "#/components/responses/bad_request"
+          },
+          "422": {
+            "$ref": "#/components/responses/validation_failed"
+          }
+        },
+        "x-github": {
+          "githubCloudOnly": false,
+          "enabledForGitHubApps": false,
+          "previews": [
+
+          ],
+          "category": "apps",
+          "subcategory": "webhooks"
+        }
+      }
+    },
+    "/app/hook/deliveries/{delivery_id}/attempts": {
+      "post": {
+        "summary": "Redeliver a delivery for an app webhook",
+        "description": "Redeliver a delivery for the webhook configured for a GitHub App.\n\nYou must use a [JWT](https://docs.github.com/apps/building-github-apps/authenticating-with-github-apps/#authenticating-as-a-github-app) to access this endpoint.",
+        "tags": [
+          "apps"
+        ],
+        "operationId": "apps/redeliver-webhook-delivery",
```

Comment:
I wonder if the operationId could be changed to something less redundant? Maybe

```suggestion
        "operationId": "apps/redeliver-webhook",
```

Reply:
👋 @rlinehan, do you have any thoughts on this?

Reply:
Hmm 🤔 . This is somewhat redundant, but I'm inclined to keep it as is for two reasons:

1. Semantically we are in fact redelivering a `delivery` not a `webhook`. So you hit `/apps/hooks/:hook_id/deliveries` to get a list of `delivery_id`, then you can get the payload from `/apps/hooks/:hook_id/deliveries/:delivery_id` and you use that `deliver_id` to trigger the redelivery with `/apps/hooks/:hook_id/deliveries/:delivery_id/attempts`

We haven’t always been super consistent in our terminology, but I do think it might be valuable to try to make more of a distinction between a webhook (the actual configuration, endpoint, etc.), and a webhook delivery (a specific delivery of a payload/event).

2. `Redeliver` is the same terminology that's been used in the UI (settings for webhook) for a long time, e.g.
<img width="941" alt="Redeliver button" src="https://user-images.githubusercontent.com/1530016/124335514-65197a80-db4f-11eb-9da2-8f1eb19bf6b0.png">

While renaming something in the UI is much easier than in the API, I think changing this terminology when it's been in place for a number of years would be opening a can of worms that I'm not entirely sure we want to open 😅 .

Reply:
Thank you for the detailed explanation 👍🏼